daphne_worker/src/dap.rs

```diff
@@ -86,37 +85,48 @@ impl<'a, D> HpkeDecrypter<'a> for DaphneWorkerConfig<D> {
             .map_err(|e| DapError::Fatal(format!("kv_store: {}", e)))?;
 
         let hpke_config_id = if keys.keys.is_empty() {
```

Comment:
Reviewer note: The main change here is to use gen_hpke_receiver_config_list to generate the HPKE keys, thereby increasing code reuse. We still only support a single KEM, but it'll now be eassier to add support for multiple KEMs in the future.